client/blocks/reader-full-post/style.scss

```diff
@@ -665,3 +665,125 @@ $reader-full-post-desktop-min: $reader-full-post-sidebar-width + $reader-full-po
 .is-reader-full-post {
 	background: var(--color-surface);
 }
+
+.reader-full-post {
+	.reader-full-post__author-block {
+		.author-compact-profile {
+			display: grid;
+			grid-template-columns: auto 1fr;
+			grid-template-areas:
+				"avatar links"
+				"avatar follow";
+			column-gap: 5px;
+			row-gap: 5px;
+			margin: 20px 0 0 0;
+			padding-right: 10px;
+			text-overflow: clip;
+			white-space: nowrap;
+
+			&:not(.has-author-icon).has-author-link {
+				grid-template-areas:
+					"links"
+					"follow";
+			}
+
+			&:not(.has-author-link):not(.has-author-icon) {
+				.reader-avatar {
+					margin: 5px 0 0 0;
+				}
+			}
+
+			@media (min-width: 1300px) {
+				display: none;
+			}
+
+			&__avatar-link {
+				grid-area: avatar;
+			}
+
+			.reader-avatar {
+				margin: 0 10px 5px 0 !important;
+				position: relative;
+				top: 2px;
+
+				@include breakpoint-deprecated( "<480px" ) {
+					position: relative;
+					top: 5px;
+				}
+
+				&:not(.has-site-and-author-icon) img {
+					margin-left: -17px;
+					top: 7px;
+				}
+
+				.gravatar {
+					height: 24px !important;
+					width: 24px !important;
+					margin-right: 1em;
+					position: relative;
+					left: 18px;
+					top: -18px;
+					vertical-align: bottom;
+					max-width: initial;
+				}
+
+				.site-icon,
+				.gridicon {
+					height: 32px !important;
```

Comment:
I'm sure this is intentional, but just making sure that these `!important`'s are absolutely needed.

Reply:
I was able to remove one `!important` by increasing the selector specificity https://github.com/Automattic/wp-calypso/pull/93816/commits/57ae25a9108953498d415d131a1b86fc79fbdd0e. The remaining selectors are overriding inline styles.